solidago/src/solidago/comparisons_to_scores/continuous_bradley_terry.py

```diff
@@ -39,6 +53,64 @@ def Delta_theta(theta_ab):
     ).sum() ** (-0.5)
 
 
+HIGH_LIKELIHOOD_RANGE_THRESHOLD = 1
```

Comment:
@lenhoanglnh Do you confirm the idea of using a Likelihood lower bound to compute the uncertainty interval, rather than a more standard 90% confidence interval?